Misc/NEWS.d/next/Documentation/2020-04-02-21-34-35.bpo-40160.8AGXVv.rst

```diff
@@ -0,0 +1 @@
+skip news
```

Comment:
Misc/NEWS entries are skipped with the `skip news` label. It's added by triagers or core devs, if the fix is deemed trivial or internal enough to warrant skipping one. If considered, this PR would qualify for a `skip news` label. Please remove this change from the PR.

Reply:
This is done.